src/core/shards/core.cpp

```diff
@@ -942,7 +961,7 @@ struct Erase : SeqUser {
 
   SHVar activate(SHContext *context, const SHVar &input) {
     const auto &indices = _indices.get();
-    if (unlikely(_isTable && _target->valueType == Table)) {
+    if (unlikely(_target->valueType == Table)) {
```

Comment:
Should we keep `unlikely` here?

If I get it correctly, it means that this shard is optimized (fast path) for sequence, and will pay the cost of a bad branch prediction for a table. Am I right?